zulip/integrations/trello/zulip_trello.py

```diff
@@ -160,8 +160,8 @@ def create_webhook(options: argparse.Namespace) -> None:
 
 def main() -> None:
     parser = argparse.ArgumentParser()
-    parser.add_argument('trello_board_name', help='The Trello board name.')
     parser.add_argument('trello_board_id', help='The Trello board short id.')
+    parser.add_argument('trello_board_name', help='The Trello board name.')
```

Comment:
@emmanuelbernard: This looks good, but I have one question, which "documentation" are you referring to? The order in [our docs for this integration](https://chat.zulip.org/integrations/doc/trello) was already reflected correctly, or am I missing something here? :)

Reply:
My apologies, I have been confused. I looked around (website and readme). You are correct the doc and the script are aligned.
I can only thing that I had id then name wired by the structure of the trello URL `https://trello.com/b/<BOARD_ID>/<BOARD_NAME>`

I'll close the PR.